src/utils/parsers.ts

```diff
@@ -184,21 +188,36 @@ const remotePaymentRequestToLocalPaymentRequest = (remotePaymentRequest: Payment
             attemptKey,
             authorisedAt,
             settledAt,
+            attemptedAmount,
             paymentMethod,
-            authorisedAmount,
             settledAmount,
+            captureAttempts,
             currency,
             walletName,
+            status,
           } = remotePaymentAttempt;
+
           localPaymentAttempts.push({
             attemptKey: attemptKey,
             occurredAt: new Date(settledAt ?? authorisedAt ?? 0),
             paymentMethod: walletName
               ? parseWalletNameToPaymentMethodType(walletName)
               : parseApiPaymentMethodTypeToLocalMethodType(paymentMethod),
-            amount: settledAmount > 0 ? settledAmount : authorisedAmount,
+            amount: attemptedAmount,
             currency: currency,
             processor: walletName ? parseApiWalletTypeToLocalWalletType(walletName) : undefined,
+            needsCapture:
```

Comment:
To determine this we can check if AuthorizedAmount in attempt is greater than SettledAmount when payment method is card.